drivers/hmis/app/graphql/mutations/create_service_type.rb

```diff
@@ -15,6 +15,11 @@ class CreateServiceType < CleanBaseMutation
     def resolve(input:)
       access_denied! unless current_user.can_configure_data_collection?
 
+      category = Hmis::Hud::CustomServiceCategory.find(input.service_category_id)
+
+      # Can't add a custom service to a HUD service category
+      access_denied! if category.service_types.any?(&:hud_service?)
```

Comment:
Let's discuss - should this be `any` or `all`?

Reply:
If I had to pick I would say "All", so we can continue supporting adding custom service types to the existing categories that have both HUD and custom (like "Check-in" that you found).

Although, I think it's simpler to just remove this check. It feels a little sticky that this logic and the picklist logic need to match (because we don't want the picklist to present an option that the mutation will reject). Nothing "bad" happens if there is a custom service type in the same category as HUD service type. - and as you pointed out we already have one like that.